easybuild/tools/module_generator.py

```diff
@@ -36,15 +36,18 @@
 import re
 import tempfile
 from vsc.utils import fancylogger
+from vsc.utils.missing import get_subclasses
 
 from easybuild.framework.easyconfig.easyconfig import ActiveMNS
-from easybuild.tools import config
-from easybuild.tools.config import build_option
+from easybuild.tools.config import build_option, get_module_syntax, install_path
 from easybuild.tools.filetools import mkdir
-from easybuild.tools.module_naming_scheme.utilities import det_hidden_modname
 from easybuild.tools.utilities import quote_str
 
 
+LUA_SYNTAX = 'Lua'
```

Comment:
move this to the class constants and call `ModuleGeneratorTcl.SYNTAX` where needed